api/github.ts

```diff
@@ -132,7 +132,7 @@ async function status ({ owner, repo, ref = 'master', context }: PathArgs) {
   const resp = await restGithub(`repos/${owner}/${repo}/commits/${ref}/status`)
 
   let state = typeof context === 'string'
-    ? resp!.statuses.filter(st => st.context.startsWith(context))
+    ? resp!.statuses.filter(st => st.context.toLowerCase().includes(context.toLowerCase()))
```

Comment:
I think we can go with `includes` but not `toLowerCase` ;)

```suggestion
    ? resp!.statuses.filter(st => st.context.includes(context))
```

Reply:
I'm exactly the opposite, haha. Can agree on using `startsWith` but not removing the `toLowerCase` :rofl: 

Services and urls should never be case sensitive. It's how such things always worked.

It's like it's `tunnckoCore`, but GitHub won't forbid them to request `https://github.com/tunnckocore` lowercased, no matter do they know how it's styled or not. It just doesn't make sense. There're always exceptions, like npm, but we should not be an exception, I don't see any reason.

Reply:
> Services and urls should never be case sensitive

Agree! It's a strong point I missed 😝